src/SFA.DAS.ASK.Data/Entities/VisitFeedback.cs

```diff
@@ -12,6 +12,7 @@ public class VisitFeedback
         public Visit Visit { get; set; }
         public FeedbackStatus Status { get; set; }
         public FeedbackAnswers FeedbackAnswers { get; set; }
+        public string IncorrectDetailsComments { get; set; }
     }
 
     public class FeedbackAnswers
```

Comment:
We'll also need a change to VisitFeedback.sql to add the new field to the table.  `IncorrectDetailsComments nvarchar(MAX)`